lib/hypernova/controller_helpers.rb

```diff
@@ -88,7 +88,7 @@ def hypernova_batch_after
           result = @hypernova_batch.submit!
           on_success(result, hash)
         rescue StandardError => e
-          on_error(e)
+          on_error(e, {}, hash)
```

Comment:
@ljharb  it seems like anything we do here short of adding a different method is a breaking change. What do you think about calling `on_error(e, nil, hash)` here? I like your suggestion as well but it would requiring a more refactoring of existing code.